yarn-project/noir-compiler/src/cli/contract.ts

```diff
@@ -20,6 +25,7 @@ export function compileContract(program: Command, name = 'contract', log: LogFn
     .option('-o, --outdir <path>', 'Output folder for the binary artifacts, relative to the project path', 'target')
     .option('-ts, --typescript <path>', 'Optional output folder for generating typescript wrappers', undefined)
     .option('-i, --interface <path>', 'Optional output folder for generating an Aztec.nr contract interface', undefined)
+    .option('--compiler <compiler>', 'Compiler to use (nargo|wasm)', 'nargo')
```

Comment:
If the default is nargo, does this mean the aztec-cli will use nargo? I can't see a change in the aztec-cli.